src/state/farms/fetchFarms.ts

```diff
@@ -12,17 +13,23 @@ const fetchFarms = async (farmsToFetch: SerializedFarmConfig[]): Promise<Seriali
   ])
 
   return farmsToFetch.map((farm, index) => {
-    const [tokenBalanceLP, quoteTokenBalanceLP, lpTokenBalanceMC, lpTotalSupply, tokenDecimals, quoteTokenDecimals] =
-      farmResult[index]
+    const [
+      tokenBalanceLP,
+      quoteTokenBalanceLP,
+      lpTokenBalanceMC,
+      lpTotalSupply,
+      [tokenDecimals],
```

Comment:
We need to take number out of array to not recignized a new key for memoized function